README.md

```diff
@@ -472,6 +472,29 @@ Prints string to stdout, and returns string with additional utility methods
 like `.to()`.
 
 
+### mktemp([options,] [templates...])
+
+Available Options:
+
++ `-d`: Create a directory instead of a file.
++ `-u`: Unsafe mode. (Delete the file before retuning). Only use if you know what you're doing.
```

Comment:
Is this what `-u` does? It says in the Linux man page that it performs a dry-run (doesn't create anything).